lib/fastly/token.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Token object
+class Fastly
+  class Token < Base
+    attr_accessor :access_token, :id, :user_id, :customer_id, :name, :last_used_at, :created_at, :expires_at, :ip, :user_agent, :sudo_expires_at, :scopes, :scope, :services, :service_id
```

Comment:
This contains extra attributes that are not documented. We should strive to make this implementation match the API as it is documented.

```suggestion
    attr_accessor :access_token, :id, :user_id, :name, :last_used_at, :created_at, :expires_at, :ip, :user_agent, :scope, :services
```

Reply:
Updated.